tests/bluetooth/controller/ctrl_cte_req/src/main.c

```diff
@@ -616,6 +616,751 @@ void test_cte_req_reject_inv_ll_param_peripheral_remote(void)
+				   struct pdu_data_llctrl_phy_req *phy_req, uint8_t events_at_start,
+				   uint8_t ctx_num_at_end)
+{
+	struct pdu_data_llctrl_phy_req rsp = { .rx_phys = PHY_1M | PHY_2M | PHY_CODED,
+					       .tx_phys = PHY_1M | PHY_2M | PHY_CODED };
+	struct pdu_data_llctrl_phy_upd_ind ind = { .instant = events_at_start +
+							      PHY_UPDATE_INSTANT_DELTA,
+						   .c_to_p_phy = phy_req->tx_phys,
+						   .p_to_c_phy = phy_req->rx_phys };
+	struct node_tx *tx = NULL;
+	struct pdu_data *pdu;
+	uint16_t instant;
+
+	/* Prepare */
+	event_prepare(&conn);
+
+	if (is_local) {
+		/* Tx Queue should have one LL Control PDU */
+		lt_rx(LL_PHY_REQ, &conn, &tx, phy_req);
+		lt_rx_q_is_empty(&conn);
+
+		/* TX Ack */
+		event_tx_ack(&conn, tx);
+
+		/* Rx */
+		lt_tx(LL_PHY_RSP, &conn, &rsp);
+
+		ind.instant += 1;
+	}
+
+	/* Done */
+	event_done(&conn);
+
+	/* Check that data tx was paused */
+	zassert_equal(conn.tx_q.pause_data, 1U, "Data tx is not paused");
+
+	if (tx != NULL) {
+		/* Release Tx */
+		ull_cp_release_tx(&conn, tx);
+	}
+	/* Prepare */
+	event_prepare(&conn);
+
+	/* Tx Queue should have one LL Control PDU */
+	lt_rx(LL_PHY_UPDATE_IND, &conn, &tx, &ind);
+	lt_rx_q_is_empty(&conn);
+
+	/* TX Ack */
+	event_tx_ack(&conn, tx);
+
+	/* Check that data tx is no lonnger paused */
+	zassert_equal(conn.tx_q.pause_data, 0U, "Data tx is paused");
+
+	/* Done */
+	event_done(&conn);
+
+	/* Save Instant */
+	pdu = (struct pdu_data *)tx->pdu;
+	instant = sys_le16_to_cpu(pdu->llctrl.phy_upd_ind.instant);
+
+	/* Release Tx */
+	ull_cp_release_tx(&conn, tx);
+
+	wait_for_phy_update_instant(instant);
+
+	check_phy_update_and_cte_req_complete(is_local, cte_req, phy_req, ctx_num_at_end);
+}
+
+/**
+ * @brief The function executes PHY update procedure in peripheral role.
+ *
+ * The main goal for the function is to run and evaluate the PHY update control procedure.
+ * In case the PHY request is remote request and there is a local CTE request then
+ * after PHY update completion CTE request is executed in the same event.
+ * In this situation the function processes verification of CTE request completion also.
+ *
+ * @param is_local        Flag informing if PHY request is local or remote.
+ * @param cte_req         Parameters of CTE request procedure. If it is NULL there were no CTE
+ *                        request.
+ * @param phy_req         Parameters of PHY update reques.
+ * @param events_at_start Number of connection events at function start.
+ * @param ctx_num_at_end  Expected number of free procedure contexts at function end.
+ */
+static void run_phy_update_peripheral(bool is_local, struct pdu_data_llctrl_cte_req *cte_req,
+				      struct pdu_data_llctrl_phy_req *phy_req,
+				      uint8_t events_at_start, uint8_t ctx_num_at_end)
+{
+	struct pdu_data_llctrl_phy_req rsp = { .rx_phys = PHY_1M | PHY_2M | PHY_CODED,
+					       .tx_phys = PHY_1M | PHY_2M | PHY_CODED };
```

Comment:
Same comment regarding common PHY values

Reply:
Changed to macro.